geonode/messaging/__init__.py

```diff
@@ -33,6 +34,8 @@
 class MessagingAppConfig(NotificationsAppConfigBase):
     name = 'geonode.messaging'
 
+    NOTIFICATIONS = (("message_recevied", _("Message received"), _("New message received in one of your threads")),)
```

Comment:
@petrus7, Little spelling error here, I think you meant "message_received" not "message_recevied".